src/pages/api/kakao-logout.ts

```diff
@@ -24,6 +24,10 @@ export default async function handler(
 ) {
   const { logoutToken } = req.body;
 
-  const tokenResponse = await getTokenFromKakao(logoutToken);
-  res.status(200).json({ tokenResponse });
+  try {
+    const tokenResponse = await getTokenFromKakao(logoutToken);
+    res.status(200).json({ tokenResponse });
+  } catch (error: any) {
```

Comment:
p3;
any 지양하면 좋을 듯 한데, Error로 하면 어떨까요~?
```suggestion
  } catch (error: Error) {
```